### PR TITLE
TRD fix ranges of plots in macro

### PR DESCRIPTION
**Pull request by bazinski**
fix the ranges of the charge windows in the plots.
q0=q1 range 256
q2 range 64 was 256

This dates back to the confusion of what was Q0 and Q2.

### CODE REVIEW

File: Detectors/TRD/reconstruction/macros/checkTrackletCharges.C [hunk 1]
```diff
@@ -23,9 +23,9 @@ using namespace o2::trd;
 std::vector<TH1F*> createTrdChargeHists()
 {
   std::vector<TH1F*> hCharges;
-  hCharges.push_back(new TH1F("Charge0", "Charge window 0;charge;counts", 64, 0, 63));
+  hCharges.push_back(new TH1F("Charge0", "Charge window 0;charge;counts", 256, 0, 255));
   hCharges.push_back(new TH1F("Charge1", "Charge window 1;charge;counts", 256, 0, 255));
-  hCharges.push_back(new TH1F("Charge2", "Charge window 2;charge;counts", 256, 0, 255));
+  hCharges.push_back(new TH1F("Charge2", "Charge window 2;charge;counts", 64, 0, 63));
```
Comment (f3sch) [inline Detectors/TRD/reconstruction/macros/checkTrackletCharges.C]: Would it make more sense to also set the ranges for Q0/1 to their respective range i.e. 0...127?

`
 hCharges.push_back(new TH1F("Charge0", "Charge window 0;charge;counts", 127, 0, 127));
 hCharges.push_back(new TH1F("Charge1", "Charge window 1;charge;counts", 127, 0, 127));
 hCharges.push_back(new TH1F("Charge2", "Charge window 2;charge;counts", 63, 0, 63));
`

Comment (bazinski) [reply Detectors/TRD/reconstruction/macros/checkTrackletCharges.C]: I thought that as well, but then its hard to see the spike on the right.

Comment (f3sch) [reply Detectors/TRD/reconstruction/macros/checkTrackletCharges.C]: Makes sense, the same argument would then hold for Q2 since we also see a spike there.

Comment (bazinski) [reply Detectors/TRD/reconstruction/macros/checkTrackletCharges.C]: yip, and your numbers above cut a bin though. Anyway apparently I need to add a copyright, not sure why only now.

**Review by f3sch — DISMISSED** [review]

**Comment by f3sch** [comment]
LGTM!

**Comment by tdietel** [comment]
The charges in Tracklet64 are 8 bits, so one could argue that a range up to 256 would be failsafe. But in practice we will only get 7/7/6 bits from the FEE, so the restricted ranges are also fine.

File: Detectors/TRD/reconstruction/macros/checkTrackletCharges.C [hunk 2]
```diff
@@ -23,9 +40,9 @@ using namespace o2::trd;
 std::vector<TH1F*> createTrdChargeHists()
 {
   std::vector<TH1F*> hCharges;
-  hCharges.push_back(new TH1F("Charge0", "Charge window 0;charge;counts", 64, 0, 63));
+  hCharges.push_back(new TH1F("Charge0", "Charge window 0;charge;counts", 256, 0, 255));
   hCharges.push_back(new TH1F("Charge1", "Charge window 1;charge;counts", 256, 0, 255));
-  hCharges.push_back(new TH1F("Charge2", "Charge window 2;charge;counts", 256, 0, 255));
+  hCharges.push_back(new TH1F("Charge2", "Charge window 2;charge;counts", 128, 0, 127));
```
Comment (tdietel) [inline Detectors/TRD/reconstruction/macros/checkTrackletCharges.C]: I think that's incorrect: the ranges should be 256, 0.0, 256.0. Or, actually, I usually prefer 256, -0.5, 255.5 to have the labels in the bin centers.

Comment (bazinski) [reply Detectors/TRD/reconstruction/macros/checkTrackletCharges.C]: oui oui, they can all potentially be 8 bit :-( However this is just plotting them, not manipulating data, so its more a question of clear easily readable graphs, no? and given that most of the time its going to be 7 7 6 when not using the scaled q values, is it not clearer to h ave 255.5 255.5 127.5 ? giving an area at the right with nothing and clearly showing the spike? Yes I definitely agree about the -0.5 to -???.5

Comment (davidrohr) [reply Detectors/TRD/reconstruction/macros/checkTrackletCharges.C]: @tdietel : could you approve, then we'Ll merge it

Comment (tdietel) [reply Detectors/TRD/reconstruction/macros/checkTrackletCharges.C]: Restricting it to 7/7/6 bits is fine. Then the ranges should be 
128, -0.5, 127.5
64, -0.5, 63.5
Please update

**Review by tdietel — CHANGES_REQUESTED** [review]

**Comment by bazinski** [comment]
@tdietel  done.

**Review by tdietel — APPROVED** [review]

**Pull request merged**